torchao/dtypes/uintx/tensor_core_tiled_layout.py

```diff
@@ -76,14 +75,9 @@ def _linear_bf16_act_uint4_weight_impl(input_tensor, weight_tensor, bias):
 
     # groupwise int4 quantization
     groupsize = weight_tensor.block_size[1]
-    if is_device(input_tensor.device.type, "cpu") and TORCH_VERSION_AT_LEAST_2_6:
-        y = torch.ops.aten._weight_int4pack_mm_for_cpu(
-            act_mat.contiguous(), packed_weight, groupsize, scale_and_zero
-        )
-    else:
-        y = torch.ops.aten._weight_int4pack_mm(
-            act_mat.contiguous(), packed_weight, groupsize, scale_and_zero
-        )
+    y = torch.ops.aten._weight_int4pack_mm(
```

Comment:
why we change `_weight_int4pack_mm_for_cpu` to `_weight_int4pack_mm`? I remember `_weight_int4pack_mm` didn't register for CPU.

Reply:
@Xia-Weiwen moved the CPU implementation to `torchao/dtypes/uintx/int4_cpu_layout.py`, where support for more activation dtypes has also been added.

This code now pertains to CUDA.

Reply:
The explanation from @sanchitintel is correct. This part of code would never be run on CPU because the layout is for CUDA only.